signbank/video/views.py

```diff
@@ -426,8 +426,18 @@ def change_glossvideo_publicity(request):
 
 def export_glossvideos_csv(request):
     """ This function will return a csv with details of published videos. No filtering the page would affect here. """
-    csv_queryset = GlossVideo.objects.filter(is_public=True).values('id', 'videofile', 'version', 'gloss__idgloss',
-                                                                    'dataset__name', 'title', 'video_type_id__english_name')
+    include_private = request.GET.get('include_private', False)
```

Comment:
**issue**: this hasn't checked that the input is a boolean value.

Because it's coming in the query string, we should expect to receive a string, and need to parse it as a boolean ourselves.

The test case will provide a boolean value, but that doesn't match the behaviour if a user has `include_private=True` in their query string